src/qt/transactiondesc.cpp

```diff
@@ -38,14 +40,19 @@ QString TransactionDesc::FormatTxStatus(const interfaces::WalletTx& wtx, const i
     else
     {
         int nDepth = status.depth_in_main_chain;
-        if (nDepth < 0)
+        if (nDepth < 0) {
             return tr("conflicted with a transaction with %1 confirmations").arg(-nDepth);
-        else if (nDepth == 0)
-            return tr("0/unconfirmed, %1").arg((inMempool ? tr("in memory pool") : tr("not in memory pool"))) + (status.is_abandoned ? ", "+tr("abandoned") : "");
-        else if (nDepth < 6)
+        } else if (nDepth == 0) {
+            QString abandoned;
```

Comment:
Using ternary operator would allow marking it as const and avoiding re-initialization. Ir this is considered more readable?

Reply:
Thanks! [Updated](https://github.com/bitcoin-core/gui/pull/313#issuecomment-841686529).